deposit/src/main/java/edu/unc/lib/deposit/fcrepo3/IngestDeposit.java

```diff
@@ -371,9 +371,7 @@ private void ingestObject(String ingestPid, boolean confirmExisting) throws Depo
 		uploadIngestFiles(foxmlDoc, pid);
 
 		// Ingest the object's FOXML
-		try {
-
-			ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+		try (ByteArrayOutputStream outputStream = new ByteArrayOutputStream()) {
```

Comment:
Should outputStream be declared as an OutputStream instead of the more specific type?